### PR TITLE
Added a reference about locales and started on text about "locale-neu…

### PR DESCRIPTION
**Pull request by aphillips**
…tral" formats.

***

### :boom: Error: 500 Internal Server Error :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on Mar 10, 2020, 10:01 PM UTC)_.

<details>
<summary>More</summary>

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [Spec Generator](https://www.w3.org/2015/labs/) - Spec Generator is the web service used to build specs that rely on ReSpec.

:link: [Related URL](https://labs.w3.org/spec-generator/?type=respec&url=https%3A%2F%2Frawcdn.githack.com%2Faphillips%2Fbp-i18n-specdev%2F32266449ace17fb4ce46e1ec20e2547e84afd321%2Findex.html%3FisPreview%3Dtrue)

```
Navigation timeout of 30000 ms exceeded
```

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20w3c/bp-i18n-specdev%2340.)._
</details>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1497,14 +1497,24 @@ <h3>Miscellaneous</h3>
...
   <li><a href="#loc_time">Working with time</a></li>
   <li><a href="#loc_names">Working with personal names</a></li>
   <li><a href="#loc_forms">Designing forms</a></li>
   <li><a href="#loc_numbers">Working with numbers</a></li>
 </ul>
 
+<section id="loc_neutral" class="subtopic">
+	<h2>Working with locale-affected values</h2>
+
+<p>Software systems that support languages and cultural preferences are said to be "internationalized". An internationalized system uses APIs to provide language or culturally specific processing, based on user preferences. These user preferences are usually referred to as a "Locale". For more information on general internationalization terminology, see <em>Language Tags and Locale Identifiers</em> [[LTLI]]</p>
```
Comment (xfq) [inline index.html]: Should "Locale" be lowercase? It is lowercase in the Abstract of [LTLI](https://www.w3.org/TR/ltli/), for example.

Comment (aphillips) [reply index.html]: But not in the title (this quotes most of the title in titlecase)

Comment (xfq) [reply index.html]: I mean the one in the inside of the quotation marks, not the one in the inside of the `em` (btw, `cite` might be better than `em` here).

Comment (aphillips) [reply index.html]: Fixed in 3226644

**Pull request merged**